src/cswinrt/code_writers.h

```diff
@@ -8450,7 +8452,13 @@ global::System.Collections.Concurrent.ConcurrentDictionary<RuntimeTypeHandle, ob
                     }
                 }),
             bind<write_class_members>(type, false, false),
-            bind<write_custom_query_interface_impl>(type));
+            bind<write_custom_query_interface_impl>(type),
+            settings.netstandard_compat ? "" : w.write_temp(R"(
```

Comment:
We can drop the netstandard check, this is a non netstandard code path.

Reply:
Ooh, cool, thank you! Done in https://github.com/microsoft/CsWinRT/pull/1505/commits/f712e4a4cc3331ca321ca9fcf196fadc832133e2.